great_expectations/core/metric_function_types.py

```diff
@@ -0,0 +1,38 @@
+import enum
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MetricFunctionTypes(enum.Enum):
+    VALUE = "value"
+    MAP_VALUES = "value"  # "map_values"
+    WINDOW_VALUES = "value"  # "window_values"
+    AGGREGATE_VALUE = "value"  # "aggregate_value"
+
+
+class MetricPartialFunctionTypes(enum.Enum):
```

Comment:
Same comment here

Reply:
@NathanFarmer I am not sure -- potentially, but it would be subject of a separate change, because this change is purely module refactoring.  Please feel to try; thanks.